python_modules/dagster-test/dagster_test/test_project/__init__.py

```diff
@@ -112,7 +112,7 @@ def find_local_test_image(docker_image):
 def build_and_tag_test_image(tag):
     check.str_param(tag, "tag")
 
-    base_python = "3.8.8"
+    base_python = "3.11"
```

Comment:
why the bump? are we breaking compat with python 3.8?

Reply:
This was just to make the test pass locally - it now matches what we run in buildkite